brainiak/utils/fmrisim.py

```diff
@@ -50,7 +50,9 @@
 Show the brain as it unfolds over time with a given opacity.
 
 
- Authors: Cameron Ellis (Princeton) 2016
+ Authors:
+ Cameron Ellis (Princeton) 2016-2017
+ Chris Boldassano (Princeton) 2016-2017
```

Comment:
Although I think this is an improved spelling, it is unfortunately "bald" not "bold"